frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/storage-review.tsx

```diff
@@ -50,6 +50,16 @@ const StorageReviewFirehose: React.FC<StorageReviewFirehoseProps> = ({
   persistentVolumeClaims,
   storageClasses,
 }) => {
+  const headers = [
+    { title: 'Name' },
+    { title: 'Interface' },
+    { title: 'Size' },
+    { title: 'Storage Class' },
+    { title: 'Access Mode' },
+    { title: 'Volume Mode' },
+    { title: 'Source' }
```

Comment:
can we keep the ordering of the Storage Tab?

Reply:
Done!